{"name": "Тест: создание курьера", "status": "failed", "statusDetails": {"message": "AssertionError: assert {'code': 409,...уйте другой.'} == {'ok': True}\n  \n  Left contains 2 more items:\n  {'code': 409, 'message': 'Этот логин уже используется. Попробуйте другой.'}\n  Right contains 1 more item:\n  {'ok': True}\n  Use -v to get more diff", "trace": "self = <test_create_courier.TestPostCreateCourier object at 0x000002002ED67020>\n\n    @allure.title('Тест: создание курьера')\n    def test_create_courier_shows_code_201(self):\n        login, password, first_name = fake_data_courier()\n        payload = {\"login\": login, \"password\": password, \"firstName\": first_name}\n        response = requests.post(f'{URL}{self.ENDPOINT}', data=payload)\n>       assert response.json() == {'ok': True}\nE       AssertionError: assert {'code': 409,...уйте другой.'} == {'ok': True}\nE         \nE         Left contains 2 more items:\nE         {'code': 409, 'message': 'Этот логин уже используется. Попробуйте другой.'}\nE         Right contains 1 more item:\nE         {'ok': True}\nE         Use -v to get more diff\n\ntests\\test_create_courier.py:18: AssertionError"}, "attachments": [{"name": "log", "source": "a9c7a6fc-d465-44aa-b989-6d964a7fa6a4-attachment.txt", "type": "text/plain"}], "start": 1717679525484, "stop": 1717679525899, "uuid": "ed4298c0-e927-457b-a710-4998ff7d67bb", "historyId": "258e4e0c5697ecd5eb5f2738651c3ae9", "testCaseId": "258e4e0c5697ecd5eb5f2738651c3ae9", "fullName": "tests.test_create_courier.TestPostCreateCourier#test_create_courier_shows_code_201", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestPostCreateCourier"}, {"name": "host", "value": "DESKTOP-USF6L0H"}, {"name": "thread", "value": "1796-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}